www-servers/lighttpd/lighttpd-1.4.53.recipe

```diff
@@ -231,10 +229,10 @@ defineDebugInfoPackage lighttpd$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
-	LIBS="-lintl" \
+	LIBS="-lintl -lgnu" \
```

Comment:
```suggestion
	LIBS="-lintl" \
```

After lighttpd is updated to a slightly newer version then `-lgnu` can be added along with `--with-attr` to the configure args